src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,476 @@
+```sql
+SELECT sum(databaseCallCount) 
+FROM Transaction
+```
+  </Side>
+  <Side>
+    <img
+  title="Select sum of the databaseCallCount from transaction query"
+  alt="A screenshot of a query selecting the sum of the databaseCallCount from transaction"
+  src={queriesnrqltutorial7}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+Finally, you can also count all recorded Transaction events. We do this with `count(*)`. In this example, we count all recorded events for all applications reporting to New Relic APM:
+```sql
+SELECT count(*) 
+FROM Transaction
+```
+  </Side>
+  <Side>
+    <img
+  title="Select all recorded transactions query"
+  alt="A screenshot of a query selecting all recorded transactions"
+  src={queriesnrqltutorial8}
+/>
+  </Side>
+</SideBySide>
+Now we're beginning to find out a lot about our data. We're measuring the boundaries of our performance, grouping event data together, and even calculating information about our chosen attribute. We've chosen duration and calculated the `average`, `max`, and `min`, added up numeric attributes, and counted all the events for a time period. Manipulating data like this will help us when we want to present data in visualizations, which assists us in spotting performance issues. Let's go to the next step: using time ranges.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="time-ranges"
+    title="3. Use time ranges"
+  >
+<SideBySide>
+  <Side>
+    Every piece of data in NRDB has a `timestamp`, and every query operates on a subset of data within a time range. If you don't provide a time range in your NRQL query, it will return the last 60 minutes by default as we saw in the previous lesson. But what if we want to see data from a different time range? How do we control the window of data we see? The answer to these questions is very straightforward: use `SINCE` and `UNTIL` keywords to provide a beginning and end to your query's time range.
+```sql 
+SELECT average(duration) 
+FROM Transaction 
+SINCE 1 day ago
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago query"
+  alt="A screenshot of a query selecting the average duration since 1 day ago"
+  src={queriesnrqltutorial9}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    We can also specify time ranges relative to when we make our queries by using keywords like `day`, `week`, `hour`, `minute`, or their plural equivalents. We can also use logical expressions like `SINCE` today or `SINCE` this week.
```

Comment:
```suggestion
    We can also specify time ranges relative to when we make our queries by using keywords like `day`, `week`, `hour`, `minute`, or their plural equivalents. We can also use logical expressions like `SINCE today` or `SINCE this week`.
```

Reply:
Thank you!